pkg/universe.dagger.io/x/berryphillips@gmail.com/onepassword/read.cue

```diff
@@ -0,0 +1,53 @@
+package onepassword
+
+import (
+	"dagger.io/dagger/core"
+	"universe.dagger.io/docker"
+)
+
+// Read a secret by secret reference
+#Read: {
+	// The account login credentials
+	credentials: #Credentials
+
+	// The reference to the secret
+	reference: #Reference
+
+	// output will contain the secret value
+	output: container.export.files["/tmp/output.txt"]
```

Comment:
I'd recommend using `core.#NewSecret` here, instead of creating a plain string.

https://github.com/dagger/dagger/blob/d4548195bb37dd07b8cdcc5b54021adf01c0814b/pkg/dagger.io/dagger/core/secrets.cue#L23-L34

Reply:
Thanks for the feedback! I've made the changes based on what I could figure out.